### ISSUE TITLE
Rules: Notifications: Add template for AssetID in URL field

### ISSUE THREAD

**Issue by DonWillems** [openremote/openremote#1464]
As a user I want to send notifications that navigate to an asset page when opened, so that my end-users immediately go to the mentioned asset.

- Check how %TRIGGER_ASSETS% and %RULESET_NAME% works
- Add %ASSET_ID% as template that places the asset id of the asset that triggered this rule. If there are multiple, use the asset id of the first asset listed on the LHS (possible?)
- Add %ASSET_IDS% as a template that places all asset ids that caused this rule to trigger comma separated
- Add these templates to the documentation

![Image](https://github.com/user-attachments/assets/9537a6d7-1819-4c0d-acdb-b71f2b5adb13)

Consider:
Adding more templates such as value, asset name, attribute name

**Comment by mihaelaaleks** [comment]
Changed solution to handle: 

- `Asset ID` template in URL: currently replaces the first asset that triggers the rule (logically the use case for the URL is linking to a single asset). 
- `Asset ID` in body: can chain Asset IDs together. 

I've tested the following scenarios (successfully): 

1. Multiple `when` conditions
2. Child/parent asset trigger for correct Asset ID

However, since I can't locally send notifications, some more extensive testing is needed.
